pyroomacoustics/open_sofa_interpolate.py

```diff
@@ -0,0 +1,556 @@
+import numpy as np
+import scipy
+import sofa
+from scipy.fft import fft,fftfreq,ifft
+import math
+import matplotlib.pyplot as plt
```

Comment:
Same here because `matplotlib` is not a hard requirement for pyroomacoustics. Please protect it as above.